Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with prometheus-operator.
+
+## Two approaches to utilizing ServiceMonitors
+What approach is the "best" is currently mostly a "personal" preference.
+
+### One ServiceMonitor for multiple Services
+You create a ServiceMonitor to target a label that is shared among the Services you want to monitor.
+This approach enables you to use less ServiceMonitors, but may require as mentioned above a shared label among all services to be monitored.
+
+### One ServiceMonitor per one Service
```

Comment:
Let's combine these two sections. And it's actually not recommended to have one `ServiceMonitor` per `Service`, only if there is no other way it is the last resort. One should always aim to have as many `Service`s underneath one `ServiceMonitor` as that will allow re-configuring to be a rare thing otherwise the chance of the Prometheus server to keep being re-configured is very high.